--- user_notifications.rb.new	2018-03-01 11:55:30.267370455 +0000
+++ user_notifications.rb	2018-03-01 11:56:00.467721062 +0000
@@ -12,18 +12,9 @@
   include Email::BuildEmailHelper
 
   def signup(user, opts = {})
-    build_email(user.email,
-                template: "user_notifications.signup",
-                locale: user_locale(user),
-                email_token: opts[:email_token])
   end
 
   def signup_after_approval(user, opts = {})
-    build_email(user.email,
-                template: 'user_notifications.signup_after_approval',
-                locale: user_locale(user),
-                email_token: opts[:email_token],
-                new_user_tips: I18n.t('system_messages.usage_tips.text_body_template', base_url: Discourse.base_url, locale: locale))
   end
 
   def notify_old_email(user, opts = {})
